app/admin/deploy_strategies.rb

```diff
@@ -23,7 +23,10 @@
       f.input :provider, as: :select, collection: DeployStrategy::PROVIDERS
       f.input :profile
       f.input :automatic
-      f.input :arguments_input, label: 'Arguments (JSON)'
+      f.input :arguments_input,
+              label: 'Arguments (JSON)',
+              hint: 'Supported properties: base (branch), head (branch), repo (e.g., org/project), " +
+              "merge_after (sec.), merge_prior_warning (sec., default 3600) slack_webhook_url, warned_pull_request_url'
```

Comment:
The list of supported arguments was getting long and complex, so I added this inline documentation.